js/main.js

```diff
@@ -6,6 +6,8 @@
           , src = Url.queryString("link") || "#"
           ;
 
+	title = (title != null) ? title : "Anything";
```

Comment:
This should be checked [here](https://github.com/IonicaBizau/anything-for-dummies/blob/gh-pages/js/main.js#L27-L28):

```js
Url.updateSearchParam("title", prompt("Title") || "Anything");
Url.updateSearchParam("link", prompt("link") || "#");
```